PORTAL/jobs/_utils.py

```diff
@@ -873,7 +874,11 @@ def resolve_cpu_arch() -> str:
 
 
 def get_termwidth(*, notty: int = 1000, unknown: int = 80) -> int:
-    if os.isatty(sys.stdout.fileno()):
+    try:
+        fileno = sys.stdout.fileno()
```

Comment:
The "magic" sys.stdout that pytest sets up raises this exception.

Reply:
lovely